tools/neura-interpreter/neura-interpreter.cpp

```diff
@@ -95,6 +95,17 @@ int main(int argc, char **argv) {
         
         valueMap[constOp.getResult()] = val;
 
+      } else if (auto phiOp = dyn_cast<neura::PhiOp>(op)) {
+        PredicatedData result{0.0f, false}; // Default to a false predicate.
+        // Find the one operand with a true predicate.
+        for (Value operand : phiOp.getOperands()) {
```

Comment:
Shouldn't we `assert` there must be **at most one** operand with predicate as `true`?

Reply:
Plz hold on, let me re-think the design. Its current form cannot support the diagram shown in README: https://github.com/tancheng/CGRA-Mapper

Reply:
I introduced `grant_predicate` and `grant_once` in https://github.com/coredac/dataflow/pull/39, so we don't need to complicate `phi`'s definition for now, so you are good to go: "assert there must be at most one operand with predicate as true".

Later on, we can introduce new op to fuse the (`phi` and `grant_once` and `const`) if needed.